src/configManager/models/database/sequelize/sequelizeConnector.js

```diff
@@ -27,25 +28,39 @@ async function updateConfig(updateValues) {
     return results;
 }
 
-async function getConfig() {
+async function deleteConfig(key) {
     const configClient = client.model('config');
-    const options = {
-        attributes: { exclude: ['updated_at', 'created_at'] }
-    };
-    const dbResults = await configClient.findAll(options);
-    const resultArr = dbResults.map(result => (result.dataValues));
+    const result = await configClient.destroy(
+        {
+            where: { key: key }
+        });
+    return result;
+}
+
+async function getConfig() {
+    let resultArr = [];
+    if (client) {
+        const configClient = client.model('config');
+        const options = {
+            attributes: { exclude: ['updated_at', 'created_at'] }
+        };
+        const dbResults = await configClient.findAll(options);
+        resultArr = dbResults.map(result => (result.dataValues));
+    }
     return resultArr;
 }
 
 async function getConfigValue(configValue) {
-    const configClient = client.model('config');
-    const options = {
-        attributes: { exclude: ['updated_at', 'created_at'] }
-    };
-    options.where = { key: configValue };
-    const dbResult = await configClient.find(options);
-    // todo: IF NOT EXISTS? throw error
-    return dbResult;
+    let dbResult = [];
+    if (client) {
+        const configClient = client.model('config');
+        const options = {
+            attributes: { exclude: ['updated_at', 'created_at'] }
+        };
+        options.where = { key: configValue };
+        dbResult = await configClient.find(options);
+    }
+    return dbResult || [];
```

Comment:
Is it possible that dbResult will be undefined ? and not empty array in case the result is empty ?

Reply:
Yes it can return null if not found